src/Controller/EnMarche/CommitteeController.php

```diff
@@ -108,6 +108,13 @@ public function unfollowAction(Request $request, Committee $committee): Response
             throw $this->createAccessDeniedException('Invalid CSRF protection token to unfollow committee.');
         }
 
+        if ($this->isGranted(CommitteePermissions::SUPERVISE, $committee)
```

Comment:
This controller already check the authorization [here](https://github.com/EnMarche/en-marche.fr/pull/1318/files#diff-7497fcbe29ed5aabeb0342f00e6db8ffR103). Please move this logic in https://github.com/EnMarche/en-marche.fr/blob/master/src/Committee/Voter/FollowCommitteeVoter.php#L47.